app/grandchallenge/reader_studies/migrations/0014_readerstudy_roll_over_answers_for_n_cases.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 3.2.10 on 2022-01-25 09:03
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reader_studies", "0013_auto_20211206_0925"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="readerstudy",
+            name="roll_over_answers_for_n_cases",
+            field=models.PositiveSmallIntegerField(
+                default=False,
+                help_text="The number of cases for which answers should roll over. For instance, if set to 1. Case 2 will start with the answers from case 1; whereas case 3 starts anew but its answers will rollover to case 4.It can be used for repeated readings with slightly different hangings, but is counterproductive when used in combination with case navigation or shuffling hangings.",
```

Comment:
```suggestion
                help_text="The number of cases for which answers should roll over. For instance, if set to 1. Case 2 will start with the answers from case 1; whereas case 3 starts anew but its answers will rollover to case 4. It can be used for repeated readings with slightly different hangings, but is counterproductive when used in combination with case navigation or shuffling hangings.",
```